docs/api/annotations.md

```diff
@@ -0,0 +1,233 @@
+# Annotations
+
+An annotation assigns pronunciation errors to specific fragments of an audio recording performed the
+an annotator user.
+
+## List all annotations
+
+A user with the 'ADMIN' role will receive all the annotations.
+Non-admins will receive only the annotations assigned to them.
+
+### URL
+
+```http
+GET /annotations HTTP/1.1
+```
+
+### Request parameters
+
+Name          | Type     | Description
+--------------|----------|------------
+`annotator`   | `string` | **Optional** The unique identier of the annotator to find the annotations for.
+`assignment`  | `string` | **Optional** The assignment for which to find the annotations.
+`recording`   | `string` | **Optional** The speech recording for which to find the annotations.
+`done`        | `bool`   | **Optional** Whether or not and how to filter on the done property. None when all values are allowed, True if only done annotations are desired. False for all not finished annotations.
+`perPage`     | `string` | **Optional** Amount of annotations to return.
+`startCursor` | `string` | **Optional** A previously returned cursor to use as an 'offset'.
+
+```http
+GET /annotations HTTP/1.1
+Accept: application/json
+```
+
+### Response
+
+The response is a JSON list with annotations.
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+[
+  {
+    "id":5646929551687680,
+    "updated":"2018-06-07T13:50:44Z",
+    "created":"2018-06-07T13:50:44Z",
+    "assignment":4873731453222912,
+    "doneBase":null,
+    "doneFull":null,
+    "recording":{
+       "recording":"5715999101812736",
+       "organisation":"aschool",
+       "challenge":"Assign_1_1_lijn3_thema9_06_EenzonvoorJet_a",
+       "digest":null,
+        "tenant":"demo"
+    },
+    "audioUrl":"https://api.itslanguage.nl/download/2257a2f8ada1446bae8d97f591169de9",
```

Comment:
Not sure but I think you are using fake data here in this response object right?
One of the latest PR's merged was changing all itslanguage.nl adresses to itslanguage.io. Perhaps this one needs to be .io as well?

Reply:
it's captured from a real response and modified a bit. Should have modified it to .io instead of nl indeed. Need to get used to our new convention ;-)